tools/packaging/common/envoy_bootstrap.json

```diff
@@ -457,7 +457,7 @@
           {
             "filters": [
               {
-                "name": "envoy.http_connection_manager",
+                "name": "envoy.filters.network.http_connection_manager",
```

Comment:
Is the test also needs to be updated?
https://github.com/istio/istio/blob/master/pkg/test/env/envoy_conf.go#L72

Reply:
It is not mandatory to update all of these instances. But good to update for consistency. My goal of this PR is to just focus on bootstrap config. Will update other places in a follow-up.